packages/tdrop-git/tdrop-git.pacscript

```diff
@@ -0,0 +1,17 @@
+name="tdrop-git"
+gives="tdrop"
+url="https://github.com/noctuid/tdrop.git"
+pkgver="0.5.0"
+homepage='https://github.com/noctuid/tdrop'
+pkgdesc="A Glorified WM-Independent Dropdown Creator"
+maintainer="Thomas Crha <thomas@9bitbyte.com>"
+makedepends=("cmake")
+depends=("xdotool" "gawk" "procps")
+
+package() {
+  sudo install -Dm755 "tdrop" "/usr/local/bin/tdrop"
+  sudo install -Dm644 "tdrop.1" "/usr/local/share/man/man1/tdrop.1"
```

Comment:
Can you compress this first before installing it?

Reply:
What do you mean by compress? Tar/zip it ? Could you show me an example of another pacscript that does what you are describing

Reply:
Or do you want to combine the command into one ?

Reply:
Sure! Basically manpages are to be compressed in the gz format with max compression. So above line 13, you would put something like:
```bash
gzip -9 tdrop.1
```
Then on line 13, you can `sudo install -Dm644 "tdrop.1.gz" -t "/usr/local/share/man/man1/"`

Reply:
I created a prepare step for the gzip compress - similar to the example.